bigquery_etl/glam/templates/extract_probe_counts_v1.sql

```diff
@@ -1,48 +1,42 @@
 {{ header }}
--- TODO: Remove deduping when dupes are fixed.
-WITH deduped AS (
-    SELECT
-        *,
-        ROW_NUMBER() OVER(
-            PARTITION BY
-                ping_type,
-                os,
-                app_version,
-                app_build_id,
-                channel,
-                metric,
-                metric_type,
-                key,
-                client_agg_type,
-                agg_type
-            ORDER BY
-                total_users DESC
-        ) AS rank
-    FROM
-        `{{ dataset }}.{{ prefix }}__view_probe_counts_v1`
-    WHERE
-        channel IS NOT NULL
-        AND app_version IS NOT NULL
-        AND total_users >= 100
-)
+
+CREATE TEMP FUNCTION udf_js_flatten(histogram ARRAY<STRUCT<key STRING, value FLOAT64>>)
+RETURNS STRING
+LANGUAGE js
+AS
+  '''
+    let obj = {};
+    histogram.map(function(r) {
+        obj[r.key] = parseFloat(r.value.toFixed(4));
+    });
+    return JSON.stringify(obj);
+''';
 
 SELECT
     channel,
     app_version as version,
-    COALESCE(ping_type, "*") as ping_type,
-    COALESCE(os, "*") AS os,
-    COALESCE(app_build_id, "*") AS build_id,
+    ping_type,
+    os,
+    app_build_id as build_id,
```

Comment:
Do these already contain the `*` instead of `NULL`?

Reply:
Yes (see [the static combination logic](https://github.com/mozilla/bigquery-etl/pull/1002/files#diff-041939f9ac853ab7f51fbe7cdead1466R445-R497)). If there are nulls in the data, it's a bug that should be resolved in the underlying view instead of patching up here.